### PR TITLE
misc: Move community forum to NR discourse

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Focus the FlowFuse employees attention to one forum, not two. Further, we can add value to the community, instead of splitting it.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by knolleary — COMMENTED** [review]
All of the urls for the new location have got duplicate `https://`

**Review by Yndira-FlowForge — CHANGES_REQUESTED** [review]
What Nick said ☝️

> All of the urls for the new location have got duplicate `https://`

**Comment by ZJvandeWeg** [comment]
@knolleary My bad, find and replace went bad. Should be fixed now.

File: src/handbook/operations/customer-success.md [hunk 1]
```diff
@@ -134,4 +134,4 @@ itself or their flows should be redirected to the
 customer direct to a repo for a 3rd party node question.
 
 [support-tickets]: ../../support/
-[support-forum]: https://community.flowfuse.com
+[support-forum]: https://https://discourse.nodered.org/c/vendors/flowfuse/24
```
Comment (knolleary) [inline src/handbook/operations/customer-success.md:137]:
```suggestion
[support-forum]: https://discourse.nodered.org/c/vendors/flowfuse/24
```

**Review by knolleary — APPROVED** [review]

**Review by Yndira-FlowForge — APPROVED** [review]

**Pull request merged**